src/utils/createReadStreamInternal.ts

```diff
@@ -324,6 +324,9 @@ export function createReadStreamInternal(
       gaxOpts,
       retryOpts,
     });
+    requestStream.on('data', () => {
+      metricsCollector.onResponse();
```

Comment:
maybe add a comment that this is for firstResponseLatency?

(This does make me wonder if we can remove this handler after it is called once for performance reasons, to avoid all those extra method calls when we only want the first one. ReadRow's on('data') is the most performance-sensitive part of the library. But maybe that's out of scope here)